rust_team_data/src/v1.rs

```diff
@@ -88,6 +88,23 @@ pub struct Lists {
     pub lists: IndexMap<String, List>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ZulipGroup {
+    pub name: String,
+    pub members: Vec<ZulipGroupMember>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
Nit: it'd be nice to have a `#[serde(rename_all = "snake_case")]` to be consistent with the rest of the API.